### PR TITLE
Fairplay ask

### PR DESCRIPTION
**Pull request by v-jizhang**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by v-jizhang** [comment]
@sanbhatt, @BrianBlum - Changed Fairplay parameter --ask from utf-8 string to hex string.

**Comment by Juliehzl** [comment]
Given all ams commands are in preview and the user number for 'az ams content-key-policy create` is not very large, it is acceptable to have such breaking change.
@v-jizhang can you help resolve conflicts and re-arrange your commands release note like other modules with [BREAKING CHANGE]?

**Comment by v-jizhang** [comment]
Solved conflicts.

**Comment by v-jizhang** [comment]
Hi all,
My last day with AMS is Monday, September 30th. Could everybody expedite the review process?

Thanks
Jinlin

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,9 +3,11 @@
 Release History
 ===============
 
-**AMS**
+**Ams**
 
-* BREAKING CHANGE: Changed job creation from "job start" to "job create".
+* [BREAKING CHANGE]
```
Comment (Juliehzl) [inline src/azure-cli/HISTORY.rst]: Can you re-edit your release note reference breaking change in HD insught?

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,9 +3,11 @@
 Release History
 ===============
 
-**AMS**
+**Ams**
```
Comment (Juliehzl) [inline src/azure-cli/HISTORY.rst]: AMS looks better.

**Comment by Juliehzl** [comment]
> Hi all,
> My last day with AMS is Monday, September 30th. Could everybody expedite the review process?
> 
> Thanks
> Jinlin

Got it! It looks good in general. Please resolve my comments and request review from me. Feel free to ping me if no response.

File: src/azure-cli/HISTORY.rst (hunk 1 above)
Comment (v-jizhang) [reply src/azure-cli/HISTORY.rst]: Edited.

Thanks
Jinlin

File: src/azure-cli/HISTORY.rst (hunk 2 above)
Comment (v-jizhang) [reply src/azure-cli/HISTORY.rst]: Changed.

Thanks
JInlin

**Comment by yonzhan** [comment]
Any update for this PR ?

**Comment by BrianBlum** [comment]
@Juliehzl , thanks

**Pull request merged**